README.md

```diff
@@ -356,11 +356,9 @@ Returns *Promise<Boolean>*.
 * **create(basename)** - creates migration, returns *Promise* with migration
 object.
 
-* **getMigrationPath(name)** - returns an absolute path of the migration executable
-file on disk by the name of the migration. Returns *Promise<String>*.
-
-* **getMigrationSourcePath(name)** - returns an absolute path of the migration source
-file on disk by the name of the migration. Returns *Promise<String>*.
+* **getMigrationPath(name, migrationFileType)** - returns an absolute path of the migration
```

Comment:
Other migration manager methods also will be affected:

* `isInitialized` should check for existing of both dirs
* `getMigrationPath` should accept `migrationFileType`